docs/contributing.md

```diff
@@ -181,6 +181,16 @@ Opsdroid is an open source chatbot framework. It is designed to be extendable, s
 
  If you don't know where to start, make sure to check the [tutorials](tutorials/introduction.md) and read the [documentation](http://opsdroid.readthedocs.io/en/latest/?badge=latest).  Remember that you can also ask for help in our [matrix channel](https://riot.im/app/#/room/#opsdroid-general:matrix.org)
 
+## Creating Consistent Logging Messages
+
+When creating log messages, we would like to see a consistency around how to structure and utilize them.
+
+* In general any copyright/legal name should be Capitalized (i.e. Slack, Telegram)
+* All usage of the opsdroid capabilities should be Capitalized (Connector, Skill, Database and Parsers)
+* Standalone sentances should not end with a period.  Use a period if there are more than one sentance in a line.
```

Comment:
I'm not sure about this one. Why should folks avoid periods in standalone sentences. When viewed as a stream of logs this may seem inconsistent?

Reply:
Mainly as each line was it's own message.   If its in a stream of logging, the newline would trigger the next thought/log message of event explanation. 

examples:
Line 1: 
Error:  running Skill dance
Line 2:
WARNING: Running Skill dance broke against name.  Check the name exists in the config file

vs 
Line 1:
Error: running Skill dance.  
Line 2:
WARNING: Running Skill dance broke against name.  Check the name exists in the config file.

I'm open to either, just kicking off the conversation.   The main thing is stating how we want it to be and stating it as such for the future.

Reply:
Sure thanks! I think I prefer consistently having full stop.s

Reply:
We might be discussing very thin grain things here but I am with Jacob on this, I would rather have punctuation on the logs. Perhaps I'm wrong to assume that a lot of people don't care much about this since if you are looking at the logs it's because you need to debug something.

Although, keeping the logs consistent and have proper punctuation (even if it's only at the end of the line), might help in making things more obvious when reading the logs in the terminal.

After spending a few minutes debugging issues with my contributions, some times it can be hard to follow the logs in debug mode, but that full stop or other punctuation will automatically signal that this log line has ended.

Reply:
I've updated my example set with the changes to be periods.    How does this look?   Ran into a unique scenario attempting to make everything a period ending sentence.  As such added a couple suggested lines to contributing doc page.